stressTest.py

```
@@ -0,0 +1,76 @@
+            '--image', autograder_image
+            ]
+            subprocess.run(command, stdout = f, stderr = f)
+            f.write(f"Submission {i} submitted \n")  
+
+            if submission_delay > 0:
+                time.sleep(submission_delay)
+    
+    sys.exit()
+
+def get_metrics(output_file):
+    if Config.LOGFILE == None:
```

Ruff

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)